app/Http/Requests/UpdateStudentProfileRequest.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace App\Http\Requests;
+
+use App\Http\Requests\Request;
+
+class UpdateStudentProfileRequest extends Request
+{
+    /**
+     * Determine if the user is authorized to make this request.
+     *
+     * @return bool
+     */
+    public function authorize()
```

Comment:
if your are sending remember token then implement authorize function here. What that remember token not found in Database?